bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBaseThingHandler.java

```diff
@@ -62,6 +62,8 @@ public SomfyTahomaBaseThingHandler(Thing thing) {
         return stateNames;
     }
 
+    private final String url = getURL();
```

Comment:
Unfortunately this will not work, the config might not be available when the handler is constructed.
```suggestion
    private final String url = "";
```

And call `url=getURL();` in `initialize();`

Reply:
fixed, thanks